diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -148,3 +155,15 @@ fun List<RulesConfig>.isRuleEnabled(rule: Rule): Boolean {
     val ruleMatched = getRuleConfig(rule)
     return ruleMatched?.enabled ?: true
 }
+
+/**
+ * Parse string into KotlinVersion
+ */
+fun String.kotlinVersion(): KotlinVersion {
+    require(this.contains("^(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)\$".toRegex()))
```

Comment:
```suggestion
    require(this.contains("^(\\d+\\.)(\\d+)\\.?(\\d+)?$".toRegex()))
```
I think it should be like this: two required sets of digits, separated by a dot, and then maybe some more digits. Also, what is `\$`?
Also, please add message to this `require`.